### PR TITLE
Stylized Basic Rules to conform with proposed table styles

### PR DESCRIPTION
**Pull request by bleehu**
We should probably hold this until we decide on table styles in the CX Design repo.

### CODE REVIEW

File: Basic Rules.txt [hunk 1]
```diff
@@ -213,29 +213,31 @@ rolls are Perception and Dexterity. The Perception stat affects a ranged weapon'
...
++--------+-------------+	
+| 10-11  |  1.5	       |	
++--------+-------------+	
+| 12-13  |  2	       |	
++--------+-------------+	
+| 14     |  2.5	       |		
++--------+-------------+
+| 15     |  3	       |		
++--------+-------------+
+
```
Comment (Turtlelord26) [inline Basic Rules.txt:240]: Something strange happened to the trailing pipes here.

**Review by Turtlelord26 — CHANGES_REQUESTED** [review]
Other than the one thing everything seems to be in order here.

File: Basic Rules.txt (hunk 1 above)
Comment (bleehu) [reply Basic Rules.txt:240]: It be proper to my eye, sir?
it be proper to my eye

Comment (Turtlelord26) [reply Basic Rules.txt:240]: Iiiiinteresting. Here's what it looks like on my screen.
image

Comment (Turtlelord26) [reply Basic Rules.txt:240]: It's just an artifact of the changes view. I went to the raw file and it's perfectly normal there.
Disregard.

**Review by Turtlelord26 — APPROVED** [review]

**Pull request merged**